ActionBuilder/package.json

```diff
@@ -37,7 +36,7 @@
   "devDependencies": {
     "@babel/core": "^7.9.0",
     "@babel/preset-env": "^7.9.0",
-    "@cjaas/common-components": "2.2.1",
+    "@cjaas/common-components": "file:../../cjaas-common-components",
```

Comment:
could we just get the common components updated and published and use the latest version instead of the static file ref?